docs/getting-started/what-is-payload.mdx

```diff
@@ -11,56 +11,107 @@ keywords: documentation, getting started, guide, Content Management System, cms,
+Payload is the Next.js fullstack framework. Write a Payload config and instantly get:
 
-<Banner type="success">
-  <strong>Payload 2.0 has been released!</strong>
-  <br />
-  Includes Postgres support, Live Preview, Lexical Editor, and more.{' '}
-  <a href="/blog/payload-2-0">Read the announcement</a>.
-</Banner>
+- A full admin panel using React server / client components, matching the shape of your data
+- Automatic database schema, including direct DB access and ownership, with migrations, transactions, proper indexing, and more
+- REST, GraphQL, and straight-to-DB Node APIs
+- Authentication which can be used in your own apps
+- A deeply customizable access control pattern
+- File storage and image management tools like cropping / focal point selection
+- Live preview - see your frontend render content changes in realtime as you update
+- Lots more
 
-Out of the box, Payload gives you a lot of the things that you often need when developing a new website, web app, or native app:
+No matter what you're building, Payload will give you backend superpowers. It can be installed in one line into any existing Next.js app, and is designed to catapult your development process. Payload takes the most complex and time-consuming parts of any modern web app and makes them simple.
 
-- A database to store your data (Postgres and MongoDB supported)
-- A way to store, retrieve, and manipulate data of any shape via full REST and GraphQL APIs
-- Authentication—complete with commonly required functionality like registration, email verification, login, & password reset
-- Deep access control to your data, based on document or field-level functions
-- File storage and access control
-- A beautiful admin UI that's generated specifically to suit your data
+It's fully open source with an MIT license and you can self-host anywhere that you can run a Node app. You can also deploy serverless to hosts like Vercel, right inside your existing Next.js app folder.
 
-## What does "headless" mean?
+Even in spite of how much you get out of the box, you still have full control over every aspect of your app - be it database, admin UI, or anything else. Every part of Payload has been designed to be extensible and customizable with modern TypeScript / React. And you'll fully understand the code that you write.
 
-A headless CMS is a system that sticks to what it's good at—managing content. It concentrates solely on granting administrators an effective way to author and maintain content, but doesn't control how and where that content is used.
+In Payload, there are no "click ops" - as in clicking around in an admin panel to define your schema. In Payload, everything is done the right way—code-first and version controlled like a proper backend. But once developers define how Payload should work, non-technical users can independently make use of its admin panel to manage whatever they need to do without having to know code whatsoever.
```

Comment:
"make use of its admin panel to manage whatever they need to do"

maybe should be:

"make use of its admin panel to manage whatever they need to"?